tools/depends/target/Makefile

```diff
@@ -97,6 +97,7 @@ pythonmodule-pil: $(ZLIB) libjpeg-turbo libpng freetype2 python26
 libsdl2: $(LINUX_SYSTEM_LIBS)
 libxslt: libgcrypt
 ffmpeg: $(ICONV) $(ZLIB) bzip2 libvorbis $(FFMPEG_DEPENDS)
+libcec: platform
```

Comment:
Ok i think i now know why the addons on our buildslaves are stale. Is it correct that any change in this file needs to trigger a rebuild of the binary addons? Atm i only track projects project/cmake for changes via git for deciding if the addons need to be rebuilt. If so - are there any other "lib" in depends which influence binary addons that i need to track for changes on jenkins?